app/(root)/posts/page.tsx

```diff
@@ -1,18 +1,80 @@
+'use client';
+
 import React from 'react';
-import Link from 'next/link';
-import { Session } from 'next-auth';
+// import Link from 'next/link';
+import { useState } from 'react';
+import { updateUser, deleteUser } from '@/lib/actions/user.actions';
+
+const Posts = (user: any) => {
+  console.log('USER', user);
+  const [data, setData] = useState({
+    id: user && user.id,
+    name: user && user.name,
+    email: user && user.email,
+  });
+
+  const handleSubmit = () => {
+    const userId = user.user.id;
```

Comment:
In the handleSubmit, the id state kept getting erased, so I needed to add this additional step here.

I imagine that when it comes time to create a real form, this will get resolved naturally